README.md

```diff
@@ -4,8 +4,12 @@
 
 This is a plugin for TinyMCE 5 that allows users to specify what language their text is written in. The plugin wraps
  the desired text in `span` tags with a `lang` attribute for the specified language. Unspecified text is assumed to be
-  written in the page's language. This helps the resulting text comply with WCAG 2.0 3.1.2 Language of Parts: "The
-   human language of each passage or phrase in the content can be programmatically determined..."
+ written in the page's language. This helps the resulting text comply with WCAG 2.0 3.1.2 Language of Parts: "The
+ human language of each passage or phrase in the content can be programmatically determined..."
+   
+The plugin is intended for use where a few words of a different language are embedded in a passage of some other
+ language. It also works when you need to change the language of the entire passage, but not as well. For best results,
```

Comment:
I think this should be changed to `... but not as well (see below Caveats section for details).`